test/files/neg/literals.scala

```diff
@@ -3,35 +3,39 @@ trait RejectedLiterals {
 
   def missingHex: Int    = { 0x }        // line 4: was: not reported, taken as zero
 
-  def leadingZeros: Int  = { 01 }        // line 6: no leading zero
+  def missingBin: Int    = { 0b }        // line 6: missing integer number
```

Comment:
I'd encourage appending to neg tests, especially to minimize the diff on the check file.

In this case, the additional manual comments in the test make this even more desirable.

I was going to try a "worksheet" style test that would update inline before/after comments, but would before be the starr or just before this commit?

Reply:
I can do that. I guess I wanted to maintain organization, but it's probably not worth it considering the diff